server.js

```diff
@@ -1,15 +1,45 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const os = require('os');
+const clc = require('cli-color');
+const crypto = require('crypto');
 
+const SIGNATURE_SECRET = 'my_secret';
 const app = express();
-app.use(bodyParser.json({ extended: true }));
 
-app.post('/webhook', express.raw({type: 'application/json'}), (request, response) => {
-    let event = request.body;
```

Comment:
Looks like you changed `/webhook` to `/`, was that to resolve confusion with having `/webhook` on the uri?